azext_iot/digitaltwins/_help.py

```diff
@@ -997,3 +997,59 @@ def load_digitaltwins_help():
           text: >
             az dt model delete-all -n {instance_or_hostname}
     """
+
+    helps["dt job"] = """
+        type: group
+        short-summary: Manage and configure jobs for a digital twin instance.
+    """
+
+    helps["dt job import"] = """
```

Comment:
will there be an export?

Reply:
No other job operations are planned as of now, but I wanted to build this in a way so that we have flexibility to add more operations if needed in future.